_episodes/08-putting-it-all-together.md

```diff
@@ -7,8 +7,11 @@ questions:
     - "Why should I use Python to create plots?"
 objectives:
     - "Import the pyplot toolbox to create figures in Python."
+    - "Use matplotlib to make further adjustments to Pandas or plotnine objects."
 keypoints:
-    - "FIXME"
+    - "Matplotlib is the engine behind plotnine and Pandas plots."
```

Comment:
How about: `Pandas and plotnine use matplotlib to create plots` ?

Reply:
I, actually, like the original version of this keypoint. So, my suggestion here is only a suggestion.